### ISSUE TITLE
Rename default config file

### ISSUE THREAD

**Issue by ngryman** [sasstools/sass-lint#533]
Hi,

I think it could be a good idea to rename or at least offer an alternative name : `.sasslintrc`.
The reason is that most tools use this convention and it feels good to know that `*rc` files are for tools :)

**Comment by DanPurdy** [comment]
I'm open to it if others are?

**Comment by Snugug** [comment]
I'm ok to allow it. Original name was following scss lint convention.

**Comment by ngryman** [comment]
Do you want a PR for this?

**Comment by DanPurdy** [comment]
If we allow it alongside .sass-lint.yml (we should) then a PR would be fine, would this file be of json syntax or would it stick with YML?

If we remove the .sass-lint.yml then it'll be a v2 thing for sure.

**Comment by ngryman** [comment]
I think that's preferable to keep both (for now at least).

`.sasslintrc` would just be an alias, so yes it should definitively have the same format. Furthermore, `yml` is perfectly fine and pretty common in the `sass` community.

**Comment by DanPurdy** [comment]
Yeah, hence why i'm keen to keep it :+1: 

I'm happy for a PR to be done. You may want to hold off until #495 is merged into develop though as it's possible that few things will change alongside it.

**Comment by ngryman** [comment]
I've got several tests failing after a fresh install: http://pastebin.com/yRVvjvvm.

I'm on `os x el capitan`
I tested with `node 4 & 5`.

Do you have any idea?

**Comment by DanPurdy** [comment]
`npm install sass-lint -g` first

**Comment by ngryman** [comment]
I'm curious, shouldn't you test against local code in `bin/sasslint.js`? How do you test your current changes to `cli`?

**Comment by DanPurdy** [comment]
Sorry i just gave you the exact wrong information!

Basically go into your project root for sass-lint and do `npm link`

this sets your local copy of sass-lint as your global one so all your changes will tested

**Comment by ngryman** [comment]
No problem :)
That said, a less tricky solution might be to simply execute `bin/sass-lint.js` in tests:
```javascript
var command = '../bin/sass-lint.js';
```

**Comment by DanPurdy** [comment]
Yeah, I think we actually started with that I can't remember why it changed..

**Comment by ngryman** [comment]
Perhaps initial versions were missing shebang. Anyway this is working now.
Thanks.

**Comment by DanPurdy** [comment]
Released in 1.12.0
